rita/src/rita_common/tunnel_manager/mod.rs

```diff
@@ -667,8 +668,10 @@ impl Handler<TunnelStateChange> for TunnelManager {
                             info!("Membership for identity {:?} is expired", msg.identity);
                             match tunnel.state {
                                 TunnelState::Registered => {
-                                    tunnel.unmonitor(make_babel_stream()?)?;
-                                    tunnel.state = TunnelState::NotRegistered;
+                                    // TODO make this real after testing
+                                    trace!("Fake removing {:?} from peers", msg.identity);
+                                    //tunnel.unmonitor(make_babel_stream()?)?;
+                                    //tunnel.state = TunnelState::NotRegistered;
```

Comment:
Hmm at least this state manipulation could be done to check state changes.

Reply:
good point